tests/ubpf_vm.rs

```diff
@@ -2420,3 +2420,18 @@ fn test_vm_tcp_sack_nomatch() {
     let vm = rbpf::EbpfVmRaw::new(Some(&prog)).unwrap();
     assert_eq!(vm.execute_program(mem.as_mut_slice()).unwrap(), 0x0);
 }
+
+#[test]
+#[should_panic(expected = "Error: unexpected call type #2 (insn #0)")]
+fn test_other_type_call(){
```

Comment:
Can we also have tests where we mess with the stack size, please? Invalid stack validation, or running out of stack space, for example.